### DISCUSSION TITLE
Enable Reply All

### DISCUSSION THREAD

**Discussion by pjpace** [simple-login/app#817] [Feature Requests]
I disagree that sending a copy to myself should be disallowed. Our non-profit organization tried using AnonAddy for our emails but have an issue with CC:s so I'm checking out SimpleLogin.

We use the service so that when you send our org an email, it goes to a team and a reply from any one person on the team will appear to come from that single address (and it's all kept anonymous). It's important that the others on the team know that the email was responded to and what the response was. AnonAddy does this just fine using reply all.

Also, it's a very common practice for people to CC: themselves to make sure the email got out and to have a second record of the response. This would be no different than that.

Can you enable self copying on an email? I'm glad to go into more details on the matter or our situation if you want - let me know if you have questions.

**Comment by nguyenkims** [comment]
We do have indeed a plan to support sending email from a mailbox to its own alias. Currently this flow isn't supported to avoid unexpected issues.
Btw apart from this "loop" email scenario, SimpleLogin should support multiple recipient emails (i.e. with CC).

**Comment by pjpace** [comment]
Thanks - seems like allowing it would be easier than declining it but I'm clearly not a programmer. :-)

I signed up for a SimpleLogin subscription for myself/my domain since that's not an issue I thought I'd encounter with my personal email... but one of the first addresses I added was the one for me and my spouse. Nope - that's the same issue; when one of us replies all, SL rejects the email that would've gone out to the other person. So I guess we reply to the third party then forward our response to the spouse so they know we responded and what we said? She's not keen on the extra steps.

Any other short term solutions people have would be great to hear.

SimpleLogin has been great for 'external' multiple recipient emails!!

**Reply by nguyenkims** [reply]
Our way of sharing a response in our team is to simply forward the response to our colleague real address (we know each other real address any way :) ) after replying to an incoming email.
